gradle-plugin/CHANGELOG.md

```diff
@@ -5,13 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 4.0.7 - 2024-01-22
+- Correctly parse git/github/gitlab urls in all environments [#419](https://github.com/EmergeTools/emerge-android/pull/419)
+
 ## 4.0.6 - 2024-01-16
 
 - Fix bug where metadata was overwriting set values. [#405](https://github.com/EmergeTools/emerge-android/pull/405)
 - Fix: logging and resolving dependencies at configuration time. [#403](https://github.com/EmergeTools/emerge-android/pull/403)
-- Move supported version classpath files to configuration time. [#394](https://github.com/EmergeTools/emerge-android/pull/394)
```

Comment:
I removed these two since they are internal only changes and would likely confuse users of our plugin.